### PR TITLE
fix: fix setStatus' getter

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#376

### CODE REVIEW

File: packages/svelte-infinitegrid/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  presets: [
+    [
+      "@babel/preset-env",
+      {
+        modules: "cjs",  
+      }
+    ]
+  ]
+};
```
Comment (WoodNeck) [inline packages/svelte-infinitegrid/babel.config.js:10]: No new line + tab & space are both used across files, please check editorconfig is working correctly.

**Review by WoodNeck — APPROVED** [review]
LGTM

File: packages/svelte-infinitegrid/src/InfiniteGrid.svelte [hunk 2]
```diff
@@ -56,13 +55,13 @@
   }
   function getLoadingElement() {
     if (hasLoadingElement) {
-      const el = container || viewer;
+      const el = container! || viewer!;
```
Comment (mixed) [inline packages/svelte-infinitegrid/src/InfiniteGrid.svelte:58]: I think I should use `!` when using `ele`. How about changing it like this?
```diff
- const el = container! || viewer!;
+ const el = container || viewer;
- return el.lastElementChild;
+ return el!.lastElementChild;
```

File: packages/svelte-infinitegrid/src/InfiniteGrid.svelte [hunk 3]
```diff
@@ -56,13 +55,13 @@
...
 
       return el.lastElementChild;
     }
   }
   function getElements() {
-    const el = container || viewer;
+    const el = container! || viewer!;
```
Comment (mixed) [inline packages/svelte-infinitegrid/src/InfiniteGrid.svelte:64]: Same here.
https://github.com/naver/egjs-infinitegrid/pull/377/files#diff-24a3c5e53ac3888efd523272eabf125d4f97733b3ba6f5dbabe7af43c5e7bcdbR58

**Review by mixed — COMMENTED** [review]
LGTM

**Review by mixed — APPROVED** [review]

**Pull request merged**